src/lib/components/SearchForm.svelte

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+    import config from '$lib/data/config';
+    import { convertStyle, s, t, themeColors } from '$lib/data/stores';
+    import { SearchIcon } from '$lib/icons';
+    import { createEventDispatcher } from 'svelte';
+
+    let phrase = '';
+    let searchbar;
+    let wholeWords = config.mainFeatures['search-whole-words-default'] ?? false;
```

Comment:
These variables actually do change in response to user input. I'll add a comment to clarify their use.

Reply:
```
let wholeWords = config.mainFeatures['search-whole-words-default'] ?? false;
let matchAccents = config.mainFeatures['search-accents-default'] ?? false;
```

These are based completely on config.

Reply:
Their initial values come from config, but they are bound to the toggle switches below, making them reactive to user input. Since the variables must be mutable for data binding to work, making these const results in a lint error.

Reply:
Oops!  My bad!